samples/bluetooth/broadcast_audio_sink/src/main.c

```diff
@@ -1044,8 +1044,8 @@ static int init(void)
 
 	usb_audio_register(hs_dev, &usb_ops);
 	err = usb_enable(NULL);
-	if (err != 0) {
-		printk("Failed to enable USB\n");
+	if (err && err != -EALREADY) {
+		printk("Failed to enable USB (%d)", err);
```

Comment:
When would it return -EALREADY? 

If something else enables USB?

Reply:
On the USB Dongle Console using USB-CDC and hence is already enabled at boot.